.github/workflows/ci.yml

```diff
@@ -17,14 +17,8 @@ jobs:
       matrix:
         # os: [ ubuntu-20.04, windows-latest, macOS-latest ]
         os: [ ubuntu-20.04 ]
-        ros_distribution: [ foxy, rolling, dashing ]
```

Comment:
Why are we removing foxy CI?

Reply:
THE RCL API change was only introduced for rolling, AFAIR.
That means master will be only compile for rolling after applying this PR

Reply:
Ok for me